app/src/main/res/menu/activity_main_drawer.xml

```diff
@@ -17,6 +17,10 @@
             android:id="@+id/nav_drawer_logout"
             android:title="@string/logout" />
 
+        <item
```

Comment:
Move above `settings` & `logout` items?

Reply:
Moved and refactored name to Create Hackathon in commit d32666441ae969c5e7a7ef9d87d3396db1105624